src/commands/logs.ts

```diff
@@ -135,37 +132,33 @@ export function printLogs(
 // TODO: unnecessary export
 export async function setupLogs(): Promise<string> {
   let projectId: string;
-  const promise = new Promise<string>((resolve, reject) => {
+  return new Promise<string>((resolve, reject) => {
     getProjectSettings().then(projectSettings => {
       console.log(`${LOG.OPEN_LINK(LOG.SCRIPT_LINK(projectSettings.scriptId))}\n`);
       console.log(`${LOG.GET_PROJECT_ID_INSTRUCTIONS}\n`);
       projectIdPrompt()
         .then(answers => {
           projectId = answers.projectId;
           const dotfile = DOTFILE.PROJECT();
-          if (!dotfile) return reject(logError(null, ERROR.SETTINGS_DNE));
+          if (!dotfile) logError(null, ERROR.SETTINGS_DNE);
           dotfile
```

Comment:
Since logError always end by calling process.exit() it should not change anything.

I opened this issue/PR because in many places of the source, the usage of logError() was (misleadingly) hinting that it would return and resume execution (which doesn’t”t occur ever)